demisto_sdk/commands/common/git_util.py

```diff
@@ -102,6 +102,52 @@ def get_commit(self, commit_or_branch: str, from_remote: bool = True) -> Commit:
                     commit_or_branch, from_remote=from_remote, exception=e
                 )
 
+    def get_previous_commit(self, commit: Optional[str] = None) -> Commit:
+        """
+        Returns the previous commit of a specific commit or the head commit / branch.
+        """
+        if commit:
+            return self.get_commit(commit, from_remote=False).parents[0]
+        try:
+            return self.repo.active_branch.commit.parents[0]
+        except TypeError:
+            return self.repo.head.commit.parents[0]
```

Comment:
add at least a debug log here with info on what happened

Reply:
narrowed it done to be more straight forward